test/util/validators_test.py

```diff
@@ -10,10 +11,16 @@
 class TestInRange(unittest.TestCase):
 
   def test_raises_if_invalid_arguments(self):
-    with self.assertRaisesRegex(ValueError, 'Must specify minimum'):
-      validators.InRange()
-    with self.assertRaisesRegex(ValueError, 'Minimum cannot be greater'):
-      validators.InRange(minimum=10, maximum=0)
+    if sys.version_info[0] < 3:
```

Comment:
This should use six instead:
https://pythonhosted.org/six/#six.assertRaisesRegex